packages/aws-client/package.json

```diff
@@ -19,11 +19,8 @@
     "url": "https://github.com/nasa/cumulus"
   },
   "scripts": {
-    "build": "true",
-    "lint": "eslint .",
-    "test": "ava",
-    "test-coverage": "nyc ava",
-    "test-ci": "npm run lint"
+    "test": "../../node_modules/.bin/ava",
```

Comment:
I want to check and make sure this is still compatible with Ava and the VScode debugger

Reply:
I verified that, if your debugger config is set up [like ava recommends](https://github.com/avajs/ava/blob/master/docs/recipes/debugging-with-vscode.md), it still works just fine.

Reply:
Yup, also tested locally and verified